components/WalletConnector/WalletConnector.tsx

```diff
@@ -0,0 +1,77 @@
+import QRCodeModal from '@walletconnect/qrcode-modal';
+import WalletConnect from '@walletconnect/client';
+import { useContext } from 'react';
+import { WalletAddressContext } from '../../contexts/WalletAddressContext';
+import { useRouter } from 'next/router';
+
+const WalletConnector = () => {
+  const { setWalletAddress } = useContext(WalletAddressContext);
+  const router = useRouter();
+  const showModal = async () => {
+    const connector = new WalletConnect({
+      bridge: 'https://bridge.walletconnect.org', // Required
+      qrcodeModal: QRCodeModal,
+    });
+
+    // Check if connection is already established
+    if (!connector.connected) {
+      // create new session
+      connector.createSession();
+    }
+
+    // Subscribe to connection events
+    connector.on('connect', (error, payload) => {
+      if (error) {
+        throw error;
+      }
+
+      // Get provided accounts and chainId
+      const { accounts, chainId } = payload.params[0];
+      if (chainId != 2828) {
```

Comment:
don't use 2828 here. It is not just a ramdom number, it is LUKSO L16 chain ID.

1) Put this number under a variable so we can **give a name** to this number.

chainId == LUKSO_L16_CHAIN_ID

is easier to understand than

chaindId == 2828

2) Put this in global conf file. You may need this number / variable in other parts of the code

Reply:
It looks like you already have the variable

https://github.com/lukso-network/wallet.universalprofile.cloud/pull/12/files#diff-e5241839cc728bcc6af8c400f43bfa3f5da78ceceb24bfeff61cbab8e49e78ceR7